src/toICN-after.js

```diff
@@ -36,10 +39,13 @@ if(detectedKey == ""){
   key = "";
   detectedKeyMinorSignature = "u";
 }
+else{
+  isKeyWritten = true;
+}
 
-let displayedKey = exports.DisplayedKey(detectedKey, detectedMinorSignature);
+let displayedKey = exports.DisplayedKey(detectedKey, detectedKeyMinorSignature);
```

Comment:
何度もすみません、mainにもう一つバグがありました・・

```suggestion
let displayedKey = exports.getDisplayedKey(detectedKey, detectedKeyMinorSignature);
```